R/AbstractPackageReporter.R

```diff
@@ -72,18 +82,31 @@ AbstractPackageReporter <- R6::R6Class(
         get_package = function(){
             return(private$packageName)
         },
-        
+      
         get_package_path = function(){
           return(private$packagePath)
         },
-        
-        get_report =  function() {
-            stop("get_report has not been implemented.")
+   
+        get_report =  function(output_file = NULL) {
```

Comment:
I think the default should be something like `file.path(getwd(), "report.html")`.

Does this actually work if you leave it as NULL?